app/client-v2/src/__tests__/stores/projectStore.spec.ts

```diff
@@ -478,5 +500,14 @@ describe("projectStore", () => {
       expect(stopPolling).toBe(true);
       expect(store.project.status).toStrictEqual({ assign: "failed", network: "failed", microreact: "failed" });
     });
+    it("should call getClusterAssignResult if assign is failed when processStatusAndGetPolling called", async () => {
+      const analysisStatus: AnalysisStatus = { assign: "failed", network: "deferred", microreact: "waiting" };
+      const store = useProjectStore();
+      store.getClusterAssignResult = vitest.fn();
+
+      await store.processStatusAndGetStopPolling(analysisStatus, "waiting");
+
+      expect(store.getClusterAssignResult).toHaveBeenCalled();
+    });
```

Comment:
Was this just a missing test or is it related to changed behaviour?

Reply:
part of changed behaviour.. we fetch assign cluster on fail as well now :smile: .. before it was just on finish